Updated sk-g1 table

pvagner opened the pull request:
Hello,

In line with the corresponding [braille-specs pull request](https://github.com/liblouis/braille-specs/pull/29) here are updates to the slovak braille table.

Both forward as well as backward translations are tested with the braille display. We'd appreciate accepting this update into the upcoming release of liblouis.

We are addressing these issues / new features with this update:

* Adding emphasis indicator op codes for bold, italic and underline.
* Fixing some back translation issues with specific multy cell symbols [discussed by email](https://www.freelists.org/post/liblouis-liblouisxml/skg1ctb-back-translation-conflict) more than 2 years ago
* Adding some additional symbols representing foreign latin alphabed characters that are sometimes used within casual life
* Fixing an issue with sum math symbol
* Slightly improving tests.
Thanks for including this.

Please let us know in case there are issues accepting these changes.

Thanks and greetings

Michal, Peter and all the Council of SABP members

bertfrees commented:
@pvagner Very nice PR, thank you! Would it be possible to add a few more tests though? A lot of things seem to have changed, but the number of added tests is somewhat limited compared to that. Or am I wrong?

pvagner commented:
@bertfrees Thanks for looking into this.
We have added some new symbols that are being standardized and fixed some issues we have discovered.
The only new feature is emphasis.

The tests we currently have are inspired from other tables that are constructed similarly i.e. I was looking at russian literary table, hungarian tables, czech table throughout the years while working on the slovak table.

If you think adding more tests is needed I will try adding them.

However can you please try to help me find a way on how to do it properly?

The easiest for me would be to add more by simply copying and pasting examples from the braille specs.
Do you think copying as much as I can is a good idea or can you please think of a better approach that might help us to find features that are not tested enough so far?

If you can agree with this I'd like to make it in a way so you can accept this update for the upcoming version.

Thanks and greetings

Peter

egli commented:
> After a bit of work I have added more tests as suggested by @bertfrees and I can push them today. Is it okay to add them here or should I create a new pull request?

You can just add them to this PR, thanks
